build/yaml/crd/nsx.vmware.com_subnetports.yaml

```diff
@@ -0,0 +1,110 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: subnetports.nsx.vmware.com
+spec:
+  group: nsx.vmware.com
+  names:
+    kind: SubnetPort
+    listKind: SubnetPortList
+    plural: subnetports
+    singular: subnetport
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: SubnetPort is the Schema for the subnetports API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: SubnetPortSpec defines the desired state of SubnetPort
+            properties:
+              VirtualMachine:
+                description: VirtualMachine labels to realize SecurityPolices (VMSelector)
+                type: string
+              subnet:
+                description: Subnet defines the attachement of the SubnetPort
+                type: string
+              subnetset:
```

Comment:
Not sure if we should set VM/Subnet/SubnetSet in ownerReference for SubnetPort CR

Reply:
I'm asking to add VM in the `ownerReferences` like what virtualnetworkinterface already did.. And I think we'd better not add multiple owners in the `ownerReferences`, which may introduce unnecessary complexity.